main/tasks/power_management_task.c

```diff
@@ -83,6 +83,15 @@ void POWER_MANAGEMENT_task(void * pvParameters)
 
         power_management->fan_rpm = Thermal_get_fan_speed(&GLOBAL_STATE->DEVICE_CONFIG);
         power_management->chip_temp_avg = Thermal_get_chip_temp(GLOBAL_STATE);
+        
+        // Get both temperatures for EMC2103
+        if (GLOBAL_STATE->DEVICE_CONFIG.EMC2103) {
+            EMC2103_temps_t temps = Thermal_get_chip_temps(GLOBAL_STATE);
+            power_management->chip_temp_avg = temps.temp1;
+            power_management->chip_temp2_avg = temps.temp2;
+        } else {
+            power_management->chip_temp2_avg = -1; // Invalid for non-EMC2103 devices
```

Comment:
I think overloading the value -1 to mean not available isn't great. Its better to have an explicit bool for if its supported, or a num_temp_sensors variable that has how many sensors and then return a list of values.